_dev/src/store/modules/accounts/actions.ts

```diff
@@ -504,6 +506,8 @@ export default {
       }
       throw new HttpClientError(response.statusText, response.status);
     }
+
+    dispatch(ActionsTypes.SAVE_WEBSITE_VERIFICATION_META, state.tokenPsAccounts);
```

Comment:
This token is used to call the API. The one you're looking for is another token you receive from an action `REQUEST_SITE_VERIFICATION_TOKEN`.

Then from my understanding, having this dispatch here is not needed as the token has been already set by the action `TRIGGER_WEBSITE_VERIFICATION_PROCESS`, called before `TRIGGER_WEBSITE_CLAIMING_PROCESS` you're modifying.
It should be moved earlier in the execution, right before the current website is already verified and claimed.